compiler/include/flags_list.h

```diff
@@ -105,11 +105,11 @@ symbolFlag( FLAG_GENERIC , npr, "generic" , "generic types, functions and argume
 symbolFlag( FLAG_GLOBAL_TYPE_SYMBOL, npr, "global type symbol", "is accessible through a global type variable")
 symbolFlag( FLAG_HAS_RUNTIME_TYPE , ypr, "has runtime type" , "type that has an associated runtime type" )
 
-// If any of the following three flags is attached to a class (resp. record or
-// union), then that type is not a Plain-Old Data (POD) type.
-symbolFlag( FLAG_HAS_USER_ASSIGNMENT , npr, "has user assignment" , "applied to classes and records for which assignment is explicitly defined")
-symbolFlag( FLAG_HAS_USER_DESTRUCTOR , npr, "has user destructor" , "applied to classes that define a destructor explicitly" )
-symbolFlag( FLAG_HAS_USER_INIT_COPY_FN , npr, "has user init copy fn" , "applied to classes that define an init copy function explicitly" )
+// Is this type a Plain-Old Data (POD) type - ie no autocopy/destructor/=
+// need ever be called - bit copies will do.
+symbolFlag( FLAG_POD , ypr, "plain old data" , "data can be bit copied")
+symbolFlag( FLAG_NOT_POD , ypr, "not plain old data" , "bit copy overridden")
+
```

Comment:
Actually, should not rely on these flags at all even post-resolution - should call isPOD(Type*) instead.